scripts/exglobal_fcst_nemsfv3gfs.sh

```diff
@@ -615,17 +685,68 @@ runSeq::
 ::
 EOF
 
+#### ww3 version of nems.configure
+if [ $cplwav = ".true." ]; then
+
+# Switch on cpl flag
+  cpl=.true.
+
+NTASKS_FV3m1=$((NTASKS_FV3-1))
+atm_petlist_bounds=" 0 $((NTASKS_FV3-1))"
+wav_petlist_bounds=" $((NTASKS_FV3)) $((NTASKS_FV3m1+npe_wav))"
+###  atm_petlist_bounds=" 0   1511"
+###  atm_petlist_bounds=$atm_petlist_bounds
+###  wav_petlist_bounds="1512 1691"
+###  wav_petlist_bounds=$wav_petlist_bounds
+  coupling_interval_sec=${coupling_interval_sec:-1800}
+  rm -f nems.configure
+cat > nems.configure <<EOF
+EARTH_component_list: ATM WAV
+EARTH_attributes::
+  Verbosity = 0
+::
+
+ATM_model:                      fv3
+ATM_petlist_bounds:             ${atm_petlist_bounds}
+ATM_attributes::
+  Verbosity = 0
+  DumpFields = false
+::
+
+WAV_model:                      ww3
+WAV_petlist_bounds:             ${wav_petlist_bounds}
+WAV_attributes::
+  Verbosity = 0
+::
+
+runSeq::
+  @${coupling_interval_sec}
+    ATM
+    ATM -> WAV
+    WAV
+  @
+::
+EOF
+fi
```

Comment:
Shouldn't there be an "else" for running the model without wave ?

Reply:
Good point, could pull lines 679 through 686 into the if-block. @ajhenrique Can you make adjustment?

Reply:
The standalone block has been moved as an else option in the cplwav if/fi block:
https://github.com/NOAA-EMC/global-workflow/blob/c72d7b5bca4a67e83916edd17fb0e5c4612a7867/scripts/exglobal_fcst_nemsfv3gfs.sh#L681-L732